doc/api/schemas/Resources.yaml

```diff
@@ -0,0 +1,2 @@
+type: object
+additionalProperties: true # Free form dictionary
```

Comment:
```suggestion
additionalProperties: {} # Free form dictionary
```

Identical to "true" but consistent with the other usages of "additionalProperties"